lib/visitor.js

```diff
@@ -6,11 +6,7 @@ const underscoreCode = "_".charCodeAt(0);
 // This Visitor API was inspired by a similar API provided by ast-types:
 // https://github.com/benjamn/ast-types/blob/master/lib/path-visitor.js
 
-function Visitor() {
-  this.visit = this.visit.bind(this);
-  this.visitWithoutReset = this.visitWithoutReset.bind(this);
-  this.visitChildren = this.visitChildren.bind(this);
-}
```

Comment:
Can you run the `reify-perf-check` tests with and without this change? I have trouble believing `Function.prototype.bind` is so slow that it should be avoided as a rule of thumb.